src/webapp/ui/webapp.js

```diff
@@ -48,140 +48,115 @@ var disableControls = function(disable) {
     appFieldset.disabled = disable;
 }
 
-var arrayIsEmpty = function(a) {
-    if (!Array.isArray(a))
-        return true;
-    if (!a.length)
-        return true;
-    if (a.length == 1 && !a[0] && a[0].length == 0)
-        return true;
-    return false;
-}
-
 var loadOptions = function() {
     var optionsElement = document.getElementById('options');
     optionsElement.innerHTML = '<label><b>Output type:</b></label>';
 
-    var availMods = Module.getAvailableModules().split(',');
-    if (arrayIsEmpty(availMods)) {
+    var availMods = Module.getAvailableModules();
+    if (availMods.size() == 0) {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/messmerd/dmf2mod/pullRequest?prid=9178335)
```suggestion
  if (availMods.size() == 0) {
```